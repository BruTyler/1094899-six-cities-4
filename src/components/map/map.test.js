import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';

const CITY = {
  defaultZoom: 1,
  location: [56, 48],
};

const APARTMENTS = [
  {
    id: 0,
    location: [56.1, 48.1],
  },
  {
    id: 1,
    location: [56.2, 48.2],
  }
];

describe(`<Map /> render suit`, () => {
  it(`<Map /> render pin list of apartments`, () => {

    const generatedTree = renderer.create(
        <Map
          apartmentList={APARTMENTS}
          city={CITY}
        />,
        {
          createNodeMock: () => {
            return document.createElement(`div`);
          }
        }
    ).toJSON();

    expect(generatedTree).toMatchSnapshot();
  });
});
