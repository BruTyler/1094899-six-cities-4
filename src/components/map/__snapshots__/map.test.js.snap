// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Map /> render suit <Map /> render pin list of apartments 1`] = `
<section
  className=" map"
/>
`;
